### DISCUSSION TITLE
docker buildx and multi-platform images

### DISCUSSION THREAD

**Discussion by timt** [sbt/sbt-native-packager#1445] [General]
Hi,

Is there a way to configure this plugin to build multi-platform docker images.

e.g. use docker buildx https://docs.docker.com/buildx/working-with-buildx/

**Comment by muuki88** [comment]
Hi,

I didn't know about buildx before. You could try to override the `dockerExecCommand`.

See https://www.scala-sbt.org/sbt-native-packager/formats/docker.html#publishing-settings

**Reply by timt** [reply]
Thanks I'll give that a try

**Reply by Fristi** [reply]
Related to #1489 , did you found a workaround?
